mne_bids/tests/test_mne_bids.py

```diff
@@ -127,6 +129,12 @@ def test_bti():
                 task=task, raw_file=raw_fname, config=config_fname,
                 hsp=headshape_fname, output_path=output_path,
                 verbose=True, overwrite=True)
-    cmd = ['bids-validator', output_path]
-    run_subprocess(cmd, shell=shell)
+
     assert op.exists(op.join(output_path, 'participants.tsv'))
+
+    # FIXME: see these issues for reference:
+    # https://github.com/mne-tools/mne-bids/pull/84
+    # https://github.com/INCF/bids-validator/issues/553
```

Comment:
can you open an issue on `mne-bids` ? This should be fixed later, right?

Reply:
I'll do that as soon as we have merged this one, so that I can refer to code with a nice link ;-)

Reply:
oh man, perfection :)